### DISCUSSION TITLE
How to search a column without adding that in list table column

### DISCUSSION THREAD

**Discussion by liza-nt** [filamentphp/filament#10057] [Help]
I have a article resource in table  i have only shown title and status and i have slug column in my articles table i want to add search for slug also but i do not want to shown in list tables is there any way to search slug with out adding in list table column
```
         ->columns([
                TextColumn::make('title')
                    ->limit(50)
                    ->searchable(),
                BadgeColumn::make('status')->colors(ArticleStatus::getStatusColors())
            ])
            ```

**Comment by leandrocfe** [comment]
You can use this
```php
->searchable(['title', 'slug'])
```

**Reply by monzer15** [reply]
can this be used with has many relation?

**Reply by leandrocfe** [reply]
You can customize the eloquent query
https://filamentphp.com/docs/3.x/tables/columns/getting-started#searching

**Reply by quasiperfect** [reply]
@liza-nt did you manage to solve this as ->searchable is not a solution

**Reply by liza-nt** [reply]
it worked for me
```
->searchable(['title', 'slug'])

**Reply by quasiperfect** [reply]
@liza-nt maybe i'm missing something but this is my code and searchable expects boolean not string or array of strings. can you post some code ?
`
public static function table(Table $table): Table
    {
        return $table
            ->searchable()
`

**Reply by liza-nt** [reply]
```
  public static function table(Table $table): Table
    {
        return $table
            ->columns([
                TextColumn::make('title')
                    ->limit(50)
                    ->searchable(['title', 'slug']),
            ])
    }

**Reply by leandrocfe** [reply]
mark as answered please

**Reply by quasiperfect** [reply]
@liza-nt is your issue so if solved mark the answer. thanks for the code i was not expecting one field to work as a search horse for the others :)

**Reply by quasiperfect** [reply]
thanks @leandrocfe
